### DISCUSSION TITLE
Make Depth mod compatible with HD

### DISCUSSION THREAD

**Discussion by JayFlowGT** [ppy/osu#29764] [Ideas] [answered]
A question/suggestion:

Why is it that these two mods are not compatible? Is this intended or a technical limitation?
Depth mod is probably my favourite of the fun mods and think it would be cool to play with both of these at the same time.
Sorry, if i missed a previous discussion about this.

**Comment by Joehuu** [comment, marked as answer]
Here's the reason from the person who implemented the mod: https://github.com/ppy/osu/pull/25653#issuecomment-1871491729

> "Hide approach circles" setting is conflicting with the hidden mod since it always has them hidden.
And even ignoring that - objects are constantly moving, so with HD enabled it would be way harder to predict their placement at the hit time.

**Reply by cl8n** [reply]
that first part highlights a problem with mod incompatibility overall imo, partial incompatibility is kind of poorly done, one optional setting shouldn't be disallowing a whole mod combo. it feels more like that setting should be incompatible with hidden... and whatever implications that has

the second point is just a different kind of challenge so I don't think that should be considered an incompatibility
